Update creating_placements.md (#9678)
braze-inc/braze-docs: _docs/_developer_guide/banners/creating_placements.md

Next, use the [`insertBanner`](https://js.appboycdn.com/web-sdk/latest/doc/modules/braze.html#insertbanner) method to replace the inner HTML of the container element.

{% alert tip %}

>  To track impressions, be sure to call `insertBanner` for `isControl`. You can then hide or collapse your container afterwards.

Reviewer: ```suggestion
To track impressions, be sure to call `insertBanner` for `isControl`. You can then hide or collapse your container afterwards.
```

> Be sure to still call `insertBanner` for `isControl` in order to track impressions. You can hide or collapse your container afterwards.

Reviewer: ```suggestion
 To track impressions, be sure to call `insertBanner` for `isControl`. You can then hide or collapse your container afterwards.
```
